scripts/psiphon_install.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```

Comment:
@juga0 I removed the install script in #425. I think it's best to only include the install script in the lepidopter distribution so we don't have to worry about it being portable here.